x-pack/plugins/secops/public/components/timeline/body/renderers/auditd_ended_session_details.test.tsx

```diff
@@ -0,0 +1,394 @@
+      const wrapper = mountWithIntl(
+        <TestProviders>
+          <AuditdEndedSessionLine
+            id="hello-i-am-an-id"
+            userName="some-user-name"
+            secondary={undefined}
+            session={undefined}
+            hostName={undefined}
+            primary={undefined}
+            processExecutable={undefined}
+            processTitle={undefined}
+            workingDirectory={undefined}
+            args={undefined}
+          />
+        </TestProviders>
+      );
+      expect(wrapper.text()).toEqual('Sessionsome-user-name');
+    });
+
+    test('it returns only a process name if only given a process name and id', () => {
+      const wrapper = mountWithIntl(
+        <TestProviders>
+          <AuditdEndedSessionLine
+            id="hello-i-am-an-id"
+            userName={undefined}
+            processExecutable="some-process-name"
+            secondary={undefined}
+            session={undefined}
+            hostName={undefined}
+            primary={undefined}
+            processTitle={undefined}
+            workingDirectory={undefined}
+            args={undefined}
+          />
+        </TestProviders>
+      );
+      expect(wrapper.text()).toEqual('Sessionended fromsome-process-name');
+    });
+
+    test('it returns only session if process title with id is given', () => {
+      const wrapper = mountWithIntl(
+        <TestProviders>
+          <AuditdEndedSessionLine
+            id="hello-i-am-an-id"
+            processTitle="some-process-title"
+            userName="some-user-name"
+            secondary={undefined}
+            session={undefined}
+            hostName={undefined}
+            primary={undefined}
+            processExecutable={undefined}
+            workingDirectory={undefined}
+            args={undefined}
+          />
+        </TestProviders>
+      );
+      expect(wrapper.text()).toEqual('Sessionsome-user-name');
+    });
+
+    test('it returns only a working directory if that is all that is given with a id', () => {
+      const wrapper = mountWithIntl(
+        <TestProviders>
+          <AuditdEndedSessionLine
+            id={'hello-i-am-an-id'}
+            workingDirectory="some-working-directory"
+            userName={undefined}
+            secondary={undefined}
+            session={undefined}
+            hostName={undefined}
+            primary={undefined}
+            processExecutable={undefined}
+            processTitle={undefined}
+            args={undefined}
+          />
+        </TestProviders>
+      );
+      expect(wrapper.text()).toEqual('Sessioninsome-working-directory');
+    });
+
+    test('it returns only the args with id if that is all that is given (very unlikely situation)', () => {
+      const wrapper = mountWithIntl(
+        <TestProviders>
+          <AuditdEndedSessionLine
+            id="hello-i-am-an-id"
+            args="arg1 arg 2 arg 3"
```

Comment:
nit: this test mixes the args with no spacing e.g. `arg1` with args that have spacing e.g. `arg 2`

Reply:
Fixed this one here and in the other areas.